database/src/migrations/20240808000000_biohub_security_rules.ts

```diff
@@ -0,0 +1,85 @@
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Species and Ecosystems Susceptible to Persecution or Harm'), 'Mineral Lick Locations', 'Security Concern Related to Mineral Lick Locations'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Species and Ecosystems Susceptible to Persecution or Harm'), 'Telemetry Hardware', 'Security Concern Related to Telemetry Hardware'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Proprietary'), 'Private Land', 'Proprietary due to Private Land'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Proprietary'), 'Time-Limited Restriction', 'Proprietary due to Time-Limited Restriction'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Proprietary'), 'First Nations Land', 'Proprietary due to First Nations Land'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Statutory Constraints'), 'Provincial Statute', 'Secured due to Provincial Statute'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Statutory Constraints'), 'Federal Statute', 'Secured due to Federal Statute'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Mule Deer Telemetry', 'Secured due to Mule Deer Telemetry'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Mule Deer Survey Data', 'Secured due to Mule Deer Survey Data'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Modeled Freshwater Fish Habitat Capability Mapping', 'Secured due to Experimental Technology'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Predator Reduction', 'Secured due to Predator Reduction Information'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Bighorn or Thinhorn Sheep', 'Secured due to Species of Sheep'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Rocky Mountain Elk Telemetry', 'Secured due to Rocky Mountain Elk Telemetry'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Rocky Mountain Elk  Survey Data', 'Secured due to Rocky Mountain Elk Survey Data'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Harvested Species', 'Secured due to Increased Risk of Illegal Harvest'),
+    (now(), (SELECT security_category_id FROM security_category WHERE name = 'Government Interests'), 'Secured Caribou Herd', 'Secured due to association with a Secured Caribou Herd'),
```

Comment:
It seems redundant to include "Secured" in the name - let's simplify the names by just removing "Secured"